packages/ui-components/src/components/alert/alert.tsx

```diff
@@ -0,0 +1,82 @@
+import { Component, Host, Prop, Event, EventEmitter, h } from '@stencil/core';
+import { EAlertType, IAlertConfig, IAlertEvents } from './alert.types';
+import { ACTION_BUTTON_TYPE, ALERT_ICON_NAMES } from './alert.config';
+import { EActionButtonType } from '../action-button/action-button.types';
+import { EComponentSize, EIconName } from '../../types';
+
+@Component({
+	tag: 'kv-alert',
+	styleUrl: 'alert.scss',
+	shadow: true
+})
+export class KvAlert implements IAlertConfig, IAlertEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) type!: EAlertType;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) size: EComponentSize = EComponentSize.Large;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) showIcon: boolean = true;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) label!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) description: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) primaryButtonLabel!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) primaryButtonIcon: EIconName;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) secondaryButtonLabel!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) secondaryButtonIcon: EIconName;
+
+	/** @inheritdoc */
+	@Event() primaryClick: EventEmitter<void>;
+	/** @inheritdoc */
+	@Event() secondaryClick: EventEmitter<void>;
+
+	private onPrimaryClick = () => {
+		this.primaryClick.emit();
+	};
+
+	private onSecondaryClick = () => {
+		this.secondaryClick.emit();
+	};
+
+	render() {
+		return (
+			<Host>
+				<div
+					class={{
+						'alert-container': true,
+						[`--type-${this.type}`]: true,
+						'small': this.size === EComponentSize.Small
+					}}
+				>
+					<div class="information">
+						{this.showIcon && <kv-icon name={ALERT_ICON_NAMES[this.type]} class="icon" />}
+						<div class="text-container">
+							<div class="title">{this.label}</div>
+							{this.description && <div class="description">{this.description}</div>}
+						</div>
+					</div>
+					<div class="buttons">
```

Comment:
Should this be more customizable? Imagine that for a new use case, we need to have a third button, we gonna add another `kv-action-button-text` here? For this purpose why not receive a list of buttons to draw with an associated configuration & callbacks for button click? In last case, having a slot here.

Reply:
Thanks for the feedback 🙏 💪 

In regards to the buttons imo both approaches are valid, I might be a bit biased toward having the component with a slot as this would allow for more flexibility and would allow us to remove the button styling override from the lib. The approach with a button array still restricts what can be drawn on the right hand side and might make it too clumsy to apply the "frankenstein" button styling, but I haven't tested that yet to draw any conclusions.

About the description... maybe we could just use slots for both? since the component might need that flexibility in the future, we could change the description and buttons to have slots and create UI specific components on the app side?

I'd like to hear the opinion of others

Reply:
I would go with slots as well. We can also think of another possibilities in the future here like having other type of buttons or components for this specific part.

Reply:
Consensus seems to be that slots are the way to go, I'll change it, if anyone objects please comment here :)